From dwhayes@gmail.com  Tue Apr 21 16:14:19 2009
From: Dave Hayes <dwhayes@gmail.com>
Date: Sat, 18 Apr 2009 02:18:37 -0500
Subject: Staging: rt2870: new device id
To: Greg KH <greg@kroah.com>
Message-ID: <1240039117.5916.1.camel@dave-hdtv>


Hey, I have an Edimax wireless USB adapter that uses the rt2870 chipset.
lsusb shows it as follows:

Bus 001 Device 002: ID 7392:7717

When I added that ID to rt2870.h, the device came up and worked as
expected.

From: Dave Hayes <dwhayes@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/rt2870/rt2870.h |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/staging/rt2870/rt2870.h
+++ b/drivers/staging/rt2870/rt2870.h
@@ -145,6 +145,7 @@
 	{USB_DEVICE(0x0789,0x0162)}, /* Logitec */		\
 	{USB_DEVICE(0x0789,0x0163)}, /* Logitec */		\
 	{USB_DEVICE(0x0789,0x0164)}, /* Logitec */		\
+	{USB_DEVICE(0x7392,0x7717)}, /* Edimax */		\
 	{ }/* Terminating entry */                      \
 }
 
